toot/tui/app.py

```diff
@@ -303,15 +305,31 @@ def async_load_instance(self):
         Attempt to update max_toot_chars from instance data.
         Does not work on vanilla Mastodon, works on Pleroma.
         See: https://github.com/tootsuite/mastodon/issues/4915
+
+        Also attempt to update translation flag from instance
+        data. Translation is only present on Mastodon 4+ servers
+        where the administrator has enabled this feature.
+        See: https://github.com/mastodon/mastodon/issues/19328
         """
         def _load_instance():
             return api.get_instance(self.app.instance)
 
         def _done(instance):
             if "max_toot_chars" in instance:
                 self.max_toot_chars = instance["max_toot_chars"]
+            if "translation" in instance:
+                # instance is advertising translation service
+                self.can_translate = Option.YES
+            else:
+                if "version" in instance and versiontuple(instance["version"])[0] < 4:
+                    # Mastodon versions < 4 do not have translation service
+                    self.can_translate = Option.NO
+
+            # translation service for Mastodon version 4.0.0-4.0.2 that do not advertise
+            # is indeterminate; as of now versions up to 4.0.2 cannot advertise
+            # even if they provide the service, but future versions, perhaps 4.0.3+
+            # will be able to advertise.
 
-        return self.run_in_thread(_load_instance, done_callback=_done)
```

Comment:
You deleted this line so we're never running above code.

Reply:
Added it back. Thanks.